Source/Scene/ModelExperimental/buildDrawCommands.js

```diff
@@ -218,17 +225,17 @@ function createWireframeIndexBuffer(primitiveRenderResources, frameState) {
 
   const vertexCount = positionAttribute.count;
   const indices = primitiveRenderResources.indices;
-  const context = frameState.context;
+
   let originalIndices;
   if (defined(indices)) {
     const indicesBuffer = indices.buffer;
     const indicesCount = indices.count;
-    const useWebgl2 = context.webgl2;
-    if (useWebgl2 && defined(indicesBuffer)) {
-      originalIndices = IndexDatatype.createTypedArray(
-        vertexCount,
-        indicesCount
-      );
+    if (defined(indicesBuffer)) {
+      const useUint8Array = indicesBuffer.sizeInBytes === indicesCount;
+      originalIndices = useUint8Array
+        ? new Uint8Array(indicesCount)
+        : IndexDatatype.createTypedArray(vertexCount, indicesCount);
```

Comment:
@lilleyse do you know why `IndexDatatype.createTypedArray` doesn't output 8-bit indices, and if there's any code that depends on it outputting 16/32 only? Or if there's any code that also has this workaround.